src/pages/room_bookings/room-booking-list-page.js

```diff
@@ -123,6 +124,12 @@ class RoomBookingListPage extends React.Component {
         return ( roomBooking.status === ReservationStatusPaid || roomBooking.status === ReservationStatusRequestedRefund) && roomBooking.amount > 0;
     }
 
+    isEditable(bookingId) {
+        const {roomBookings} = this.props;
+        let roomBooking = roomBookings.find(rb => rb.id === bookingId);        
+        return (roomBooking.status === ReservationStatusPaid || roomBooking.status === ReservationStatusFree);
```

Comment:
ReservationStatusFree does not exists ... a free reservations is Paid and roomBooking.amount == 0